### DISCUSSION TITLE
Selecting exact frequency

### DISCUSSION THREAD

**Discussion by EliasA97** [jgromes/RadioLib#582] [General]
Looking at SX1262.cpp in order to find how frequency setting works, I found this: 

![image](https://user-images.githubusercontent.com/56277356/193637891-3773e58c-56eb-4f25-be11-426913e23460.png)

It seems that there are specific values that are chosen whatever frequency we select. 

The fact is that the ebyte SX1262 module at 433Mhz supports a frequency range from 410~493 MHz. 
Let's say I want to use the specific frequency of 434.79 Mhz. According to the .cpp file, the library will choose the 430Mhz value. 
Is that true? Also is there any way to choose the exact frequency value that we need?

**Comment by jgromes** [comment]
> It seems that there are specific values that are chosen whatever frequency we select.

The code you show only performs frequency calibration, which is done in rough bands. The actual frequency configuration is performed in the `setFrequencyRaw` method, which should be precise down to 1 Hz.
